src/ApproxManifoldProducts.jl

```diff
@@ -1,5 +1,6 @@
 module ApproxManifoldProducts
 
+using Base: print_stackframe
```

Comment:
I think this slipped in by mistake?

Reply:
huh, not sure what that is.  Will remove thanks